meta-mentor-staging/recipes-support/curl/curl_%.bbappend

```diff
@@ -1,3 +0,0 @@
-EXTRA_OECONF_append_feature-mentor-staging = "\
-    --with-ca-path=${sysconfdir}/ssl/certs \
-"
```

Comment:
Is this something that's handled with the with-ca-bundle switch?
Ref: https://git.openembedded.org/openembedded-core/tree/meta/recipes-support/curl/curl_7.74.0.bb#n58

Reply:
As far as I can tell they're independent, but their defaults are based on `--sysconfdir`. I confirmed that a build of curl has ca-path set correctly with or without the argument, presumably based on the configure.ac's defaults.